lib/log_file.c

```diff
@@ -132,13 +148,14 @@ int32_t
 qb_log_file_reopen(int32_t t, const char *filename)
 {
 	struct qb_log_target *target = qb_log_target_get(t);
+	int rc;
 
 	if (filename) {
 		(void)strlcpy(target->filename, filename, PATH_MAX);
```

Comment:
Isn't changing the filename field premature here? If `open()` fails, we keep the old `FILE` structure after all. I overlooked this added complication; looks like one must pass the new file name to `_do_file_reload()` in a separate argument...

Reply:
I believe you. It might be useful to close this avenue of future confusion anyway.